vpt/tests/test_gaze_detector.py

```diff
@@ -0,0 +1,73 @@
+import sys
+# If you know better way than using this sys.path thingy, tell me
```

Comment:
This seems like the way it should be, although it can be a little bit cleaner by moving it to an `__init__.py` file as described in [this SO answer](https://stackoverflow.com/a/31489441). Also, don't you think the tests folder should be outside `vpt`? It is not supposed to be a distributable part of the package, and maybe that is part of the problem?

There also seems to be a [command line argument](https://docs.python.org/3/library/unittest.html#test-discovery) to set the starting directory for discovery